descriptors/ModuleDescriptor-template.json

```diff
@@ -2,6 +2,22 @@
   "id": "@project.artifactId@-@project.version@",
   "name": "Camnuda BPM Module",
   "provides": [
+    {
+      "id": "camunda-history",
```

Comment:
These additions do not appear to be related to the referenced community sprint issue [MODCAMUNDA-16](https://folio-org.atlassian.net/browse/MODCAMUNDA-16).

Reply:
@jeremythuff can we get an issue for exposing these endpoints?